### ISSUE TITLE
aarch64 

### ISSUE THREAD

**Issue by landaisbenj** [Apipa169/Assistant-Relay-for-Hassio#73]
**Is your feature request related to a problem? Please describe.**
I try to install this add-on on my arm64 virtual machine based on qemu64 (freebox delta french provider).

**Describe the solution you'd like**
An aarch64 version :)

**Describe alternatives you've considered**
I don't have alternative :( but if you want, i can test something :)

**Additional context**
![image](https://user-images.githubusercontent.com/32144431/111908949-7b497200-8a5b-11eb-800e-d8da3b739f6f.png)
## System Health

version | core-2021.3.4
-- | --
installation_type | Home Assistant Supervised
dev | false
hassio | true
docker | true
virtualenv | false
python_version | 3.8.7
os_name | Linux
os_version | 4.19.0-14-arm64
arch | aarch64
timezone | Europe/Paris

<details><summary>Hass.io</summary>

host_os | Debian GNU/Linux 10 (buster)
-- | --
update_channel | stable
supervisor_version | supervisor-2021.03.6
docker_version | 19.03.13
disk_total | 78.7 GB
disk_used | 23.3 GB
healthy | true
supported | true
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.2.0), Let's Encrypt (4.11.0), FTP (4.0.1), AirCast (3.0.0), Home Assistant Google Drive Backup (0.103.1), Duck DNS (1.12.5), Visual Studio Code (3.1.0), Node-RED (8.1.2), Terminal & SSH (9.1.0), AdGuard Home (4.0.0), Bitwarden RS (0.8.0), SQLite Web (3.0.1), InfluxDB (4.0.3)

</details>

**Comment by landaisbenj** [comment]
I see all of that
https://github.com/Apipa169/Assistant-Relay-for-Hassio/issues/3#issuecomment-558994764I see all of this:
https://github.com/Apipa169/Assistant-Relay-for-Hassio/issues/28#issuecomment-600728640

https://github.com/Apipa169/Assistant-Relay-for-Hassio/issues/30#issuecomment-614256615
https://github.com/Apipa169/Assistant-Relay-for-Hassio/issues/31#issuecomment-616656026
https://github.com/Apipa169/Assistant-Relay-for-Hassio/issues/33#issuecomment-651843625
x'C

**Comment by landaisbenj** [comment]
@Apipa169 Did you have the possibility to make an aarch64 version or not ?

**Comment by Apipa169** [comment]
Not at this time. I'm quite busy lately and this will not be the first priority. I hope you understand!

**Comment by landaisbenj** [comment]
Yes I understand 😊 but if sometimes you have the time .... 😄 maybe we can let this open if another dev saw my need... ?

**Comment by Apipa169** [comment]
Sure! Any contributions are welcome.

**Comment by landaisbenj** [comment]
Thanks.
